templates/corporate/zephyr-mirror.html

```diff
@@ -5,30 +5,45 @@
 {% block portico_content %}
     <h1 style="margin-top:30px;">The Zephyr mirror script (not recommended)</h1>
 
-<p>Webathena is the easiest way to use Zulip.  With Webathena, you don't need
-to run any software in a screen session.  Your Athena password never leaves
-your computer and the only access that any Zulip computer ever receives is the
-ability to send/receive zephyrs as you.</p>
-
-<p>We recommend that you use Webathena instead of following these instructions.
-Please only follow these instructions if you know that you want to run the zephyr
-mirror script instead of using Webathena.</p>
-
-<h3 id="mirror">Running the Zephyr mirror script</h3>
-    <p>On an Athena dialup
-    (<a href="https://linerva.mit.edu">linerva.mit.edu</a>,
-     <a href="https://athena.dialup.mit.edu">athena.dialup.mit.edu</a>,
-    etc.), run the Zephyr mirroring script inside a screen
-    session.</p>
-
-    <p><strong>IMPORTANT: You must keep the Zephyr mirroring script
-    always running in a session with unexpired Kerberos
-    tickets.</strong>  The script forwards personals you receive on
-    Zephyr to Zulip, and forwards messages you send in Zulip to
-    Zephyr.  <strong>If it is not running, only other Zulip users
-    will receive messages you send from Zulip!</strong>.  The Zulip
-    web application will warn you if the Zephyr mirroring script is
-    not running.</p>
+    <p>
+        Webathena is the easiest way to use Zulip.  With Webathena, you don't need
+        to run any software in a screen session.  Your Athena password never leaves
+        your computer and the only access that any Zulip computer ever receives is the
+        ability to send/receive zephyrs as you.
+    </p>
+
+    <p>
+        We recommend that you use Webathena instead of following these instructions.
+        Please only follow these instructions if you know that you want to run the zephyr
+        mirror script instead of using Webathena.
+    </p>
+
+    <h3 id="mirror">Running the Zephyr mirror script</h3>
+
+    <p>
+        On an Athena dialup
+        (<a href="https://linerva.mit.edu">linerva.mit.edu</a>,
+        <a href="https://athena.dialup.mit.edu">athena.dialup.mit.edu</a>,
+        etc.), run the Zephyr mirroring script inside a screen
+        session.
+    </p>
+
+    <p>
+        <strong>
+            IMPORTANT: You must keep the Zephyr mirroring script
+            always running in a session with unexpired Kerberos
+            tickets.
+        </strong>
+        The script forwards personals you receive on
+        Zephyr to Zulip, and forwards messages you send in Zulip to
+        Zephyr.
+        <strong>
+            If it is not running, only other Zulip users
+            will receive messages you send from Zulip!
+        </strong>.
```

Comment:
This is also a user-visible change, adding a space between “! .”. In this case both versions are wrong, of course, since there should only be one punctuation mark.